Tests/VaporFluentTests/CacheTests.swift

```diff
@@ -0,0 +1,52 @@
+import XCTest
+@testable import VaporFluent
+
+class CacheTests: XCTestCase {
+    func testHappyPath() throws {
+        // config specifying memory database
+        var config = Config([:])
+        try config.set("fluent.driver", "memory")
```

Comment:
It might be nice at some point, even after release if we took out some of these strings for our defaults, so it'd be `try config.set(.fluentDriver, .memory)` not sure what it would take just been thinking about it a bit lately.